docs/ipc/usage.md

```diff
@@ -94,12 +94,11 @@ This command only shows subnets that have been registered to the gateway, i.e. t
 
 * To join a subnet with the `ipc-cli`
 ```bash
-./bin/ipc-cli subnet join --subnet <subnet-id> --collateral <collateral_amount> --public-key <public_key_validator_addr>
+./bin/ipc-cli subnet join --subnet <subnet-id> --collateral <collateral_amount>
```

Comment:
If the `--from` argument is missing like here, I would expect it to fallback to the default address, but in the code below I don't see that's what happens :S